conda/plugins/manager.py

```diff
@@ -202,6 +203,16 @@ def get_solver_backend(self, name: str | None = None) -> type[Solver]:
 
         return backend
 
+    def get_auth_handler(self, name: str) -> type[AuthBase] | None:
+        """
+        Get the auth handler with the given name or None
+        """
+        auth_handlers = self.get_hook_results("auth_handlers")
+        matches = tuple(item for item in auth_handlers if item.name == name.strip())
```

Comment:
For both the solver and subcommands we only compare `str.lower()` names, do we want to do the same here?

Reply:
Yeah, that makes sense.